features/manage-position/Create.tsx

```diff
@@ -96,7 +96,7 @@ const Create = () => {
       // We want to round down the number for better UI display, but we don't actually want the collateral
       // amount to round down since we want the minimum amount of collateral to pass the GCR constraint. So,
       // we'll add a tiny amount of collateral to avoid accidentally rounding too low.
-      setCollateral((minBackingCollateral + 0.00005).toFixed(4));
+      setCollateral(Math.ceil(minBackingCollateral).toString());
```

Comment:
@eng pls double check my math. If you CEIL the backing collateral, then the GCR check should always succeed right?

Reply:
how many decimal points will this be `CEIL`ing to?

Reply:
0. If we don't care about preserving precision then we can completely avoid the GCR errors that manifest itself as "gas required exceeds allowance"

Reply:
I see, that makes sense! SGTM

Reply:
can you give an example of what these values should be? I don't understand how this can even work without big numbers.

Reply:
This seems okay to me... but I think ceiling to the nearest 1BTC or 1 WETH kinda defeats the point of these buttons. If we're ceiling to the nearest full unit of collateral, the max and min buttons seem like they add complexity to the dapp without adding a lot of value.

It seems like all of these issues are basically coming out of the fact that we're doing very high precision math with plain JS floating point numbers. Is it difficult to move all of these to BNs? Or is that super annoying?

Reply:
Oh shoot I didn't even think about the fact that 1 BTC and 1 WETH are very large numbers. This seems like a good fix for the `tokens` side of things, but not the collateral